tests/lib/rules/max-depth.js

```diff
@@ -11,7 +11,7 @@
 var rule = require('../../../lib/rules/max-depth.js'),
   RuleTester = require('eslint').RuleTester;
 
-const ruleTester = new RuleTester({ parserOptions: { ecmaVersion: 6 } });
+const ruleTester = new RuleTester({ parserOptions: { ecmaVersion: 9 } });
```

Comment:
you don't need to change this for the entire test file; you can pass `parserOptions: { ecmaVersion: 9 } }` directly in the test case that needs it.